webapp/src/components/form/FormDate.js

```diff
@@ -34,19 +38,20 @@ class FormDate extends React.Component {
             {this.props.description ? (
               <FormToolTip description={this.props.description} />
             ) : (
-              <div />
-            )}
+                <div />
+              )}
           </div>
 
-        <DateTimePicker
-          id={this.props.id}
-          ref={input => {
-            this.dateInput = input;
-          }}
-          onChange={this.onChange}
-          value={this.props.value ? new Date(this.props.value) : new Date()}
-          format="y-MM-dd"
-        />
+          <DateTimePicker
+            id={this.props.id}
+            ref={input => {
+              this.dateInput = input;
+            }}
+            className={this.props.id && this.props.errorText.length > 0 ? "react-datetime-picker error" : "react-datetime-picker"}
```

Comment:
Please change `this.props.id && this.props.errorText.length` to `this.props.id && this.props.errorText && this.props.errorText.length` condition, incase errorText is ever passed in as empty. Other than that this looks all good.